commit 417e89690d618700e13846d31b0ca8a5a2badba5
Author: Zach DeCook <zachdecook@librem.one>
Date:   Sun Aug 18 20:46:29 2024 -0400

    Revert "cmake: link libOpenGL instead of legacy libGL (#44)"
    
    This reverts commit 9b33a38f86135db1b26f21d8581d512dbe106549.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 05e6b38..0db5891 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -66,7 +66,7 @@ target_include_directories(
   PRIVATE "./src" "./src/include" "./protocols" "${CMAKE_BINARY_DIR}")
 set_target_properties(aquamarine PROPERTIES VERSION ${AQUAMARINE_VERSION}
                                             SOVERSION 7)
-target_link_libraries(aquamarine OpenGL::EGL OpenGL::OpenGL PkgConfig::deps)
+target_link_libraries(aquamarine OpenGL::EGL OpenGL::GL PkgConfig::deps)
 
 check_include_file("sys/timerfd.h" HAS_TIMERFD)
 pkg_check_modules(epoll IMPORTED_TARGET epoll-shim)
